lib/ansible/modules/cloud/cloudstack/cs_rootpassword.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# (c) 2018, Gregor Riepl <onitake@gmail.com>
+# based on cs_sshkeypair (c) 2015, René Moser <mail@renemoser.net>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible. If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: cs_rootpassword
```

Comment:
I would prefer to have a different name because technically it is not limited to root user, it could be a different user (or a diffferent OS family)

One of these would be ok:
- cs_instance_pw_reset?
- cs_instance_password_reset?
- cs_instance_password?

Reply:
Good point.
More suggestions, since it will set the password of the "default" user:
* cs_defaultpassword
* cs_instance_defaultpassword
* cs_password_reset
* cs_password

Reply:
`cs_instance_password_reset` (or `cs_instance_pw_reset`) would be great as it clearly states what the module does. Cloudstack also supports updating host passwords.

Reply:
I renamed the module to `cs_instance_password_reset`, that should be as descriptive as it gets.